charts/incubator/smtp-relay/values.yaml

```diff
@@ -0,0 +1,57 @@
+image:
+  repository: tccr.io/truecharts/smtp-relay
+  pullPolicy: IfNotPresent
+  tag: v0.5.1@sha256:CHANGE_ME
+
+securityContext:
+  container:
+    runAsNonRoot: false
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        port: 25
+
+smtprelay:
+  networks:
+    - 172.16.0.0/16
+  domains: []
+  generic:
+    address: ""
+    port: 587
+    user: ""
+    pass: ""
+    aliases: ""
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              type: tcp
+            readiness:
+              type: tcp
+            startup:
+              type: tcp
+          env:
+            PORT: "{{ .Values.service.main.ports.main.port }}"
+            RELAY_NETWORKS: '{{ join " : " .Values.smtprelay.networks }}'
+            RELAY_DOMAINS: '{{ join " : " .Values.smtprelay.domains }}'
+            DISABLE_IPV6: true
+            # # Generic SMTP Relay
+            SMARTHOST_ADDRESS: "{{ .Values.smtprelay.generic.address }}"
+            SMARTHOST_PORT: "{{ .Values.smtprelay.generic.port }}"
+            SMARTHOST_USER: "{{ .Values.smtprelay.generic.user }}"
+            SMARTHOST_PASSWORD: "{{ .Values.smtprelay.generic.pass }}"
+            SMARTHOST_ALIASES: '{{ join " : " .Values.smtprelay.generic.aliases }}'
```

Comment:
This is joining a list but on Line 29 aliases its a string?

Reply:
uhh my prs are mixed